test/test_helm_python_django_app.py

```diff
@@ -7,13 +7,35 @@
 
 test_dir = Path(os.path.abspath(os.path.dirname(__file__)))
 
+VERSION = os.getenv("VERSION")
+IMAGE_NAME = os.getenv("IMAGE_NAME")
+OS = os.getenv("TARGET")
```

Comment:
I'd add `lower` to be sure that the OS will match the keys in the dict TAGS.

```suggestion
OS = os.getenv("TARGET").lower()
```